tests/lib/rules/dropdown-needs-arialabeledby-and-label-v9.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const RuleTester = require("eslint").RuleTester;
+
+const rule = require("../../../lib/rules/dropdown-needs-arialabeledby-and-label-v9");
+
+RuleTester.setDefaultConfig({
+    parserOptions: {
+        ecmaVersion: 6,
+        ecmaFeatures: {
+            jsx: true
+        }
+    }
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+ruleTester.run("dropdown-needs-arialabeledby-and-label-v9", rule, {
+    valid: [
+        `<><Label id={comboId}>Best pet</Label> <Dropdown aria-labelledby={comboId} multiselect={true} placeholder="Select an animal" {...props} > {options.map((option) => ( <Option key={option} disabled={option === "Ferret"}> {option} </Option> ))}</Dropdown></>`,
```

Comment:
Could we also test if the id is a string value? e.g.` <><Label id="my-dropdownid" /><Dropdown aria-labelledby="my-dropdownid" /></>`